pulsar-broker/pom.xml

```diff
@@ -199,6 +198,12 @@
     </dependency>
 
     <dependency>
+      <groupId>com.github.zafarkhaja</groupId>
+      <artifactId>java-semver</artifactId>
+      <version>0.9.0</version>
```

Comment:
For consistency, we define all the dependencies version in the top level pom, in the `<dependencyManagement>` section